### PR TITLE
Add first sum method

### PR DESCRIPTION
**Pull request by natacardona**

### CODE REVIEW

File: Calculator/calculator.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def sum(num1,num2):
+    return num1+num2
+
+#Testing sum
+seven = sum(1,6)
```
Comment (jlpuma24) [inline Calculator/calculator.py]: Just curious:

What is the result of this call?

`sum("1", "6")`

This method allow numbers or strings? Can you validate please?

**Review by jlpuma24 — CHANGES_REQUESTED** [review]

File: Calculator/calculator.py (hunk 1 above)
Comment (natacardona) [reply Calculator/calculator.py]: This is done, I already add the changes

Comment (jlpuma24) [reply Calculator/calculator.py]: Ok this is good but.. This method only allow `int` values? What about `float` values? And also what about negative values? What is the result of `sum(-1, 6)` is this ok?

**Review by jlpuma24 — CHANGES_REQUESTED** [review]
Thanks for the latest commit, this is good but.. This method only allow int values? What about float values? And also what about negative values? What is the result of sum(-1, 6) is this ok?

File: Calculator/calculator.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+def sum(num1,num2):
+    # Check if the variable is an integer
```
Comment (jlpuma24) [inline Calculator/calculator.py]: Is this ok? Are you only validating integer values? Can you change this to: 

`# Check if the input values are numbers`

File: Calculator/calculator.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    if (isinstance(num1, int) and isinstance(num2,int)) or (isinstance(num1, float) and isinstance(num2,float)):
+        return num1+num2
+    else:
+        print("This method not allow strings.")
+    
+
+#Testing sum
+seven = sum(1,6)
+test_string = sum("1", "6")
+test_float = sum(1720,232)
```
Comment (jlpuma24) [inline Calculator/calculator.py]: Do you test: `sum(1.3, 2)` is possible?

**Review by jlpuma24 — CHANGES_REQUESTED** [review]

File: Calculator/calculator.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+        return num1+num2
+    else:
+        print("This method not allow strings.")
+    
+
+#Test cases
+seven = sum(1,6)
+test_string = sum("1", "6")
+test_float = sum(1720,232)
+test_float_and_number = sum(1, 2.3)
```
Comment (jlpuma24) [inline Calculator/calculator.py]: Do you test: `sum(-1, 2.3)`?

**Review by jlpuma24 — CHANGES_REQUESTED** [review]

**Review by jlpuma24 — APPROVED** [review]
👍 you're g2g, merge!

**Pull request merged**